modules/compute/gke-node-pool/variables.tf

```diff
@@ -319,3 +319,14 @@ variable "specific_reservation" {
     values = null
   }
 }
+
+variable "host_maintenance_interval" {
+  description = "Specifies the frequency of planned maintenance events."
+  type        = string
+  default     = ""
```

Comment:
AFAIU, Terraform doesn't omit empty strings but only nulls. If the default is an empty string, for a blueprint that doesn't provide the relevant setting the `host_maintenance_policy` block would be included in the generated plan, unless the condition in the /gke-node-pool/main.tf#167 checks for valid values.

Reply:
Fixed the conditional, so this will not be included unless not ""